hw/xbox/nv2a/nv2a.h

```diff
@@ -446,21 +444,16 @@ void pgraph_init(NV2AState *d);
 void *pfifo_puller_thread(void *opaque);
 void pgraph_destroy(PGRAPHState *pg);
 void update_irq(NV2AState *d);
-
-
 void pgraph_context_switch(NV2AState *d, unsigned int channel_id);
 void pgraph_wait_fifo_access(NV2AState *d);
 void pgraph_method(NV2AState *d,
                    unsigned int subchannel,
                    unsigned int method,
                    uint32_t parameter);
 
-#define DEFINE_PROTO(prefix)                                              \
-    uint64_t prefix##_read(void *opaque, hwaddr addr, unsigned int size); \
-    void prefix##_write(void *opaque,                                     \
-                        hwaddr addr,                                      \
-                        uint64_t val,                                     \
-                        unsigned int size);
+#define DEFINE_PROTO(n)                                              \
+    uint64_t n##_read(void *opaque, hwaddr addr, unsigned int size); \
+    void n##_write(void *opaque, hwaddr addr, uint64_t val, unsigned int size);
```

Comment:
nit: This is shorter but also slightly less readable. I think the `prefix` name helped beginners to understand what goes on here. `n` is a lot more cryptic. I'd rather shorten or remove the argument names.